public/javascripts/case-progress.js

```diff
@@ -35,11 +32,15 @@
     return noteEventListener
   }
 
-  const setupAutoSave = (textarea) => {
-    textarea.addEventListener('keypress', getAutoSaveHandler(textarea))
+  const setupAutoSaveHearingNote = (textarea) => {
+    textarea.addEventListener('keypress', getAutoSaveHandler(textarea,  'summary/auto-save-new-note', (textarea, event) => ({ note: event.srcElement.value, hearingId: textarea.dataset.hearingid }) ))
```

Comment:
is ```keypress``` the correct event here? I know previously I've used the ```keyup``` event (which stops multiple events firing if someone has a key held down)

Reply:
Updated